modules/API/API.js

```diff
@@ -715,25 +717,33 @@ function initCommands() {
                     mode: JitsiRecordingConstants.mode.STREAM,
                     streamId: youtubeStreamKey || rtmpStreamKey
                 };
-            } else {
-                logger.error('Invalid recording mode provided');
```

Comment:
We don't need the print anymore?

Reply:
No because if there is no recording the mode won't be set, just the transcription flag.